src/ranged.cpp

```diff
@@ -1003,9 +1038,17 @@ static int print_ranged_chance( const player &p, const catacurses::window &w, in
         }
 
         auto hotkey = front_or( type.action.empty() ? "FIRE" : type.action, ' ' );
-        print_colored_text( w, point( 1, line_number++ ), col, col,
-                            string_format( _( "<color_white>[%s]</color> %s: Moves to fire: <color_light_blue>%d</color>" ),
-                                           hotkey, label, moves_to_fire ) );
+        if( ( panel_type == "compact" || panel_type == "labels-narrow" ) && display_type != "numbers" ) {
+            print_colored_text( w, point( 1, line_number ), col, col, string_format( _( "%s %s:" ), label,
+                                aim_l ) );
+            right_print( w, line_number++, 1, c_light_blue, _( "Moves" ) );
+            right_print( w, line_number, 1, c_light_blue, string_format( ( "%d" ), moves_to_fire ) );
```

Comment:
```suggestion
            right_print( w, line_number, 1, c_light_blue, string_format( "%d", moves_to_fire ) );
```

Reply:
Thanks for the review. It all looks good and very helpful.